Fix the inaccuracy in the number of IR elements to be removed

qwe661234 opened the pull request:
In the `libc_substitute`, only one IR needs to be removed, but the original version removes two. Therefore, the error of NULL pointer dereference occurs when running doom.

jserv commented:
> see #255

Enhance the descriptions and clarify the necessary actions, which may involve either deprecating the current pull request or treating the suggested modifications as a separate and distinct request.

qwe661234 committed:
Fix the inaccuracy in the number of IR elements to be removed

In the libc_substitute, only one IR needs to be removed, but the original
version removes two. Therefore, the error of NULL pointer dereference
occurs when running doom.
